README.md

```diff
@@ -214,6 +214,24 @@ parcel in your invoice that points to the same blob but with a `requires` condit
 the handler group. It also creates parcels for any parcels that the handler `requires`
 in its original bindle.
 
+### Reusable library modules
+
+To create a 'library' parcel that can be referenced as an external handler, provide a
```

Comment:
extremely nitty nit: I think it would be `an` even though this is mention, not use.